### PR TITLE
fix: runner startup crash

### PR DESCRIPTION
**Pull request by WinPlay02**
### Summary of Changes

- fixes a crash on startup as the multiprocessing manager causes a new process to spawn during the bootstrap phase
- this error only occurred during execution of the main.py with a new interpreter

### CODE REVIEW

File: tests/safeds_runner/server/test_websocket_mock.py [hunk 1]
```diff
@@ -15,6 +17,12 @@
     message_type_runtime_error,
     message_type_runtime_progress,
 )
+from safeds_runner.server.pipeline_manager import PipelineManager
+
+
+@pytest.fixture(scope="session", autouse=True)
+def _prepare_tests() -> None:
+    safeds_runner.server.main.app_pipeline_manager = PipelineManager()
```
Comment (lars-reimann) [inline tests/safeds_runner/server/test_websocket_mock.py:25]: This is quite ugly. It's also problematic that `PipelineManager` must never be initialized on the global level of a module - a restriction that must be documented carefully.

I'd instead suggest removing the problematic code from the `__init__` method of the `PipelineManager`. Instead, lazily initialize the fields such as `_multiprocessing_manager` when they are needed, like when `execute_pipeline` is called. You can turn the attributes into [`cached_properties`](https://docs.python.org/3/library/functools.html#functools.cached_property) for this purpose.

File: src/safeds_runner/server/main.py
[the remark this answers is not in the document]
Comment (lars-reimann) [reply src/safeds_runner/server/main.py:1]: Maybe have a look at how CLI tests for the `library-analyzer` work: https://github.com/Safe-DS/Library-Analyzer/blob/main/tests/library_analyzer/cli/test_cli.py

[the remark this answers is not in the document]
Comment (lars-reimann) [reply src/safeds_runner/server/main.py:1]: #16 added a script to launch the runner. You can use that to spawn a subprocess (command `safe-ds-runner`).

File: tests/safeds_runner/server/test_websocket_mock.py (hunk 1 above)
Comment (WinPlay02) [reply tests/safeds_runner/server/test_websocket_mock.py:25]: The message queue thread still needs to be started at some point.
Should this also be delayed until the first call to `execute_pipeline` was made?

Comment (lars-reimann) [reply tests/safeds_runner/server/test_websocket_mock.py:25]: Is there some downside to delaying it?

Comment (WinPlay02) [reply tests/safeds_runner/server/test_websocket_mock.py]: There should be none.

**Review by lars-reimann — APPROVED** [review]
Nice, thanks for the fast fix.

**Pull request merged**